src/state/slices/assetsSlice/assetsSlice.ts

```diff
@@ -105,17 +123,18 @@ export const assets = createSlice({
   initialState,
   reducers: {
     clear: () => initialState,
-    upsertAssets: (state, action: PayloadAction<UpsertAssetsPayload>) => {
-      state.byId = Object.assign({}, state.byId, action.payload.byId) // upsert
-      state.ids = Array.from(new Set(state.ids.concat(action.payload.ids)))
+    upsertNonFungibleAssets: (state, action: PayloadAction<UpsertAssetsPayload>) => {
+      state.nonFungible.byId = Object.assign({}, state.nonFungible.byId, action.payload.byId) // upsert
+      state.nonFungible.ids = Object.keys(state.nonFungible.byId)
     },
-    upsertAsset: (state, action: PayloadAction<Asset>) => {
+    upsertNonFungibleAsset: (state, action: PayloadAction<Asset>) => {
```

Comment:
We should still be able to fetch fungibles - which we do when e.g upserting UNI-V2 assets data, or asset description on this same file on l.170:

https://github.com/shapeshift/web/blob/25c6ece1c1c628f4392594095a8f9413d8eb0a10/src/state/slices/assetsSlice/assetsSlice.ts#L170